api/src/pcapi/routes/backoffice_v3/finance/blueprint.py

```diff
@@ -13,6 +13,8 @@
 from pcapi.core.bookings import models as bookings_models
 from pcapi.core.finance import exceptions as finance_exceptions
 import pcapi.core.finance.models as finance_models
+from pcapi.core.history import models as history_models
+import pcapi.core.history.api as history_api
```

Comment:
```suggestion
from pcapi.core.history import api as history_api
```
Même s'il y a les deux formes, essaye d'écrire les deux que tu ajoutes de la même manière :)